yt_dlp/extractor/cda.py

```diff
@@ -173,18 +175,36 @@ def extract_format(page, version):
                     video['file'] = video['file'].replace('adc.mp4', '.mp4')
             elif not video['file'].startswith('http'):
                 video['file'] = decrypt_file(video['file'])
-            f = {
+            c_quality = video.get('quality')
+            qualities = video.get('qualities')
+            for q in qualities:
+                if qualities[q] == c_quality:
+                    c_quality = q
```

Comment:
```suggestion
            c_quality = next((k for k, v in qualities.items() if v == c_quality), c_quality)
```
also give a proper name for `c_quality`. maybe `video_quality`?